Exiled.API/Features/Roles/Scp173Role.cs

```diff
@@ -43,19 +43,24 @@ internal Scp173Role(Scp173GameRole baseRole)
             ObserversTracker = scp173ObserversTracker;
 
             if (!SubroutineModule.TryGetSubroutine(out Scp173BlinkTimer scp173BlinkTimer))
-                Log.Error("Scp173BlinkTimer not found in Scp096Role::ctor");
```

Comment:
if you want you can deleted all of them now i have see nobody complaing about seing this error they was initially here just to make sure than all of them work

Reply:
just in case I prefer to leave it, for future errors or something, but yeah, if you want, I can delete it

Reply:
Okay